plugin-api/src/main/java/org/hyperledger/besu/plugin/data/Transaction.java

```diff
@@ -136,4 +136,11 @@
    * @return if present, the message call data
    */
   Optional<? extends UnformattedData> getData();
+
+  /**
+   * Returns the transaction payload.
```

Comment:
I think it's worth articulating how this field relates to `getData()` and `getInit()`.  Maybe something like:

>The data payload of this transaction.  

>If this transaction is a message-call to an account (the {@link #getTo} field is present), this same value will be exposed by {@link #getData}.  If instead this is a contract-creation transaction (the {@link #getTo}  field is absent), the payload is also exposed by {@link #getInit}.

Reply:
Nice suggestion.